Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py

```diff
@@ -1638,10 +1638,7 @@ def url_command(client: Client, score_calculator: ScoreCalculator, args: dict, r
     execution_metrics = ExecutionMetrics()
     for url in urls:
         try:
-            raw_response = client.url(
-                url, relationships
-            )
-            demisto.results(raw_response)
```

Comment:
Removing this line is a breaking change. You used to return the raw_response and you won't return it anymore. please specify this in the release notes.

Reply:
Thanks for your comments, actually this line never worked as expected, the `raw_response` dict hasn't a `contents` key. I'm removing to avoid unnecessary errors and because consistency, as the rest of IOC commands doesn't have it.